### PR TITLE
Add flag for sending LastFM replies.

### PR DESCRIPTION
**Pull request by Nedlinin**
This will let us turn this off easily while we debug some of the replies but still give us the other bot functionality (post thresholds)

### CODE REVIEW

**Comment by Nedlinin** [comment]
I'd like to see us merge this and then go ahead and run Blotter for the weekend or so with SEND_ARTIST_REPLIES and REMOVE_SUBMISSIONS set to false.

If that goes well we can turn on REMOVE_SUBMISSIONS while we debug some of the issues arising from LastFM lookups of the artists bio.

File: config.py [hunk 1]
```diff
@@ -26,6 +28,7 @@ def __init__(self):
             self.USER_NAME = config['REDDIT']['USER_NAME']
             self.SUBREDDIT = config['REDDIT']['SUBREDDIT']
             self.REMOVE_SUBMISSIONS = config['REDDIT']['REMOVE_SUBMISSIONS']
+            self.SEND_ARTIST_REPLY = config['REDDIT']['SEND_ARTIST_REPLY']
```
Comment (martijnboers) [inline config.py]: Can you add it to the `config.ini.dist`?

**Comment by martijnboers** [comment]
Looking at your commits: Pycharm community is really nice, can highly recommend it

**Comment by Nedlinin** [comment]
> Looking at your commits: Pycharm community is really nice, can highly recommend it

Believe it or not, I'm using IntelliJ Ultimate with the Python plugin.  

I suspect the reality is my inexperience with Python and the plugin performing just *slightly* differently than the JS and Java ones I'm used to.

File: config.py (hunk 1 above)
Comment (Nedlinin) [reply config.py]: Done!

**Pull request merged**